plugins/org/src/components/Cards/Group/MembersList/MembersListCard.tsx

```diff
@@ -106,14 +107,14 @@ const MemberComponent = (props: { member: UserEntity }) => {
             }}
             textAlign="center"
           >
-            <Typography variant="h5">
+            <Typography variant="h6">
               <Link
                 to={generatePath(
                   `/catalog/:namespace/user/${metaName}`,
                   entityRouteParams(props.member),
                 )}
               >
-                {displayName}
+                <OverflowTooltip text={displayName} />
               </Link>
             </Typography>
             {profile?.email && (
```

Comment:
curious if you think we should change the email to use `OverflowTooltip` for consistency, or keep it where it shows the entire email on hover?

Reply:
Oh hmm, I guess the entire email is always accessible via the link either way, so could probably switch it over to use `OverflowTooltip` as well? I think that'd make sense

Reply:
I agree! Also allows us to remove some custom CSS for it. Just changed it